components/nodemanager-service/pgdb/migration/sql/02_add_unique_to_tags.up.sql

```diff
@@ -1,5 +0,0 @@
-ALTER TABLE IF EXISTS tags
-  ADD UNIQUE (key, value);
-
-ALTER TABLE IF EXISTS nodes_tags
-  ADD UNIQUE (node_id, tag_id);
```

Comment:
I don't think we can just delete a migration that's been applied to all the nodes in our infrastructure can we?

I think we might need to:

1. edit the migration here to either not add the uniqueness constraint or do nothing
1. add another migration that if the table and constraint exist, delete them

This change was deployed already and deleting the code that deployed it won't un-deploy it.

Reply:
ya, i was gonna manually do that for our envs but it's probably safer to do it in the migration. ill push a file up

Reply:
(in case anyone other than us is consuming dev)

Reply:
We've also got about 15 deployed environments now. It's probably easier here.